stories/icon-tab-bar/icon-tab-bar.stories.js

```diff
@@ -0,0 +1,1434 @@
+                                    <span class="fd-list__title">Subsection 1.1</span>
+                                </a>
+                            </li>
+                            <li tabindex="-1" class="fd-icon-tab-bar__line-separator"></li>
+                            <li tabindex="-1" role="listitem" aria-level="3" class="fd-list__item fd-list__item--link fd-icon-tab-bar__list-item">
+                                <a tabindex="0" class="fd-list__link">
+                                    <span class="fd-list__title">Subsection 1.1.1</span>
+                                    <span class="fd-icon-tab-bar__badge"></span>
+                                </a>
+                            </li>
+                            <li tabindex="-1" class="fd-icon-tab-bar__line-separator"></li>
+                            <li tabindex="-1" role="listitem" aria-level="4" class="fd-list__item fd-list__item--link fd-icon-tab-bar__list-item">
+                                <a tabindex="0" class="fd-list__link">
+                                    <span class="fd-list__title">Subsection 1.1.1.1</span>
+                                </a>
+                            </li>
+                            <li tabindex="-1" class="fd-icon-tab-bar__line-separator"></li>
+                            <li tabindex="-1" role="listitem" aria-level="5" class="fd-list__item fd-list__item--link fd-icon-tab-bar__list-item">
+                                <a tabindex="0" class="fd-list__link">
+                                    <span class="fd-list__title">Subsection 1.1.1.1.1</span>
+                                </a>
+                            </li>
+                            <li tabindex="-1" class="fd-icon-tab-bar__line-separator"></li>
+                            <li tabindex="-1" role="listitem" aria-level="1" class="fd-list__item fd-list__item--link fd-icon-tab-bar__list-item">
+                                <a tabindex="0" class="fd-list__link">
+                                    <span class="fd-list__title">Subsection 2</span>
+                                </a>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+            </li>
+            <li role="presentation" class="fd-icon-tab-bar__item">
+                <a role="tab" class="fd-icon-tab-bar__tab" href="#" id="tab6">
+                    <span class="fd-icon-tab-bar__tag">Section 3</span>
+                </a>
+            </li>
+        </ul>
+    </div>
+</div>
+`;
+singleClick.storyName = 'Single Click Icon Tab Bar';
+singleClick.parameters = {
+    docs: {
+        storyDescription: `When there is only one click area per tab (also including tabs with sub-items), regular tabs get selected immediately on after the click is released. Tabs with sub-items trigger the expand of a menu (Popover) showing its sub-items. The parent tab title remains displayed when its child is select.
```

Comment:
Rewrite: `When there is only one click area per tab (also including tabs with sub-items), regular tabs get selected immediately after the click is released. Tabs with sub-items trigger the expansion of a menu (Popover) showing its sub-items...